tasks/main.yml

```diff
@@ -69,6 +69,12 @@
     state: present
   with_flattened: "{{ airflow_plugins|map(attribute='required_pips')|list }}"
 
+- name: Force install SQLAlchemy==1.13.15
```

Comment:
shouldn't this be done after installing airflow with force-reinstall?

Reply:
Yeah you're right, was looking at the steps above that install the requirements but sqlalchemy comes after that I guess.